tests/test_config_checker.py

```
@@ -978,38 +983,77 @@ def test_check_config_missing_ble_address(
+
+        self.assertTrue(result)
+
+    @patch("mmrelay.cli.parse_arguments")
+    @patch("mmrelay.cli.get_config_paths")
+    @patch("os.path.isfile")
+    @patch("builtins.open", new_callable=mock_open)
+    @patch("mmrelay.cli.validate_yaml_syntax")
+    @patch("builtins.print")
+    def test_check_config_valid_host_ipv4(
+        self,
+        mock_print,
```

Ruff (0.14.13)

Unused method argument: `mock_print`

(ARG002)